File: PHP/car.php
<?php
require_once('account.php');
class Car{
    public $id;
    public $license;
    public $driver;
    public $passenger;

    public function __construct($license, $driver) {
        $this->license = $license;
        $this->driver = $driver;
      }
    
      public function PrintDataCar(){
        echo "license: $this->license, conductor: {$this->driver->name}, document: {$this->driver->document}";
      }

      public function getPassenger() {
        return $this->passenger;
    }


    public function setPassenger($passenger) {
        if($passenger == 4){
            $this->passenger = $passenger;
        }else{
            echo "Necesitas asignar 4 pasajeros! ";
        }
        
    }


    public function getId() {
        return $this->id;
    }


    public function setId($id) {
        $this->id = $id;
    }


    public function getLicense() {
        return $this->license;
    }


    public function setLicense($license) {
        $this->license = $license;
    }


    public function getDriver() {
        return $this->driver;
    }


    public function setDriver($driver) {
        $this->driver = $driver;
    }
}
?>

File: Python/car.py
from account import Account


class Car:
    id = int
    license = str
    driver = Account("", "")
    passenger = int

    def __init__(self, license, driver):
        self.license = license
        self.driver = driver

    def printDataCar(self):
        if self.passenger is not None:
            print("License: " + self.license + " Name Driver: " +
                self.driver.name + " Pasajeros: " + str(self.passenger))


    def getPassenger(self):
        return self.passenger


    def setPassenger(self, passenger):
        if passenger == 4:
            self.passenger = passenger
        else:
            print("Necesitas asignar 4 pasajeros!")


    def getId(self):
        return self.id


    def setId(self, id):
        self.id = id


    def getLicense(self):
        return self.license


    def setLicense(self, license):
        self.license = license


    def getDriver(self):
        return self.driver


    def setDriver(self, driver):
        self.driver = driver
